src/Alert/Alert.js

```diff
@@ -9,6 +9,13 @@ import CloseIcon from '../icons/CloseIcon'
 
 import { colors } from '../variables'
 
+// We need this on every component to apply our minimal base
+// This runs on top of normalize.css which is included here in the storybook
+// template, we don't want to call it from here since we don't want it included
+// on every component
+import globalStyles from '../styles.global'
+globalStyles()
```

Comment:
Yes good point, I'm going to try to define it on every component instead and see if that resets the outside styles then.